packages/cli-core/API.md

```diff
@@ -25,6 +27,9 @@ export enum COLOR {
     RED = "31m"
 }
 
+// @public (undocumented)
+export type DependencyType = 'dev' | 'prod';

```

Comment:
not sure how this type ended up here but it should disappear from this api.

Reply:
Done d14222187c